underfs/hdfs/src/main/java/alluxio/underfs/hdfs/HdfsUnderFileSystemFactory.java

```diff
@@ -32,6 +35,7 @@
 @ThreadSafe
 public final class HdfsUnderFileSystemFactory implements UnderFileSystemFactory {
 
+  private static final Logger LOG = LoggerFactory.getLogger(HdfsUnderFileSystemFactory.class);
```

Comment:
nit: move this up a line. except at the start of the file we always have a newline before starting a new function.

Reply:
actually I don't see this `Logger` is used

Reply:
Removed the Logger. Was left in from debugging.